index.html

```diff
@@ -14,25 +14,32 @@
 
 <body>
     <section id="app">
+        <context-menu ref="menu"></context-menu>
         <config-menu></config-menu>
-        <header id="main-header" class="margin-bottom" style="height:100px">
+        <header id="main-header" class="margin-bottom" style="height:100px" v-on:contextmenu.prevent="$refs.menu.openContextMenu">
             <navbar v-bind:suites="suites"></navbar>
         </header>
 
-        <main v-bind:class="{ 'full-size': !AppStatus.config.bottomBar }">
+        <main v-bind:class="{ 'full-size': !AppStatus.config.bottomBar }" v-on:contextmenu.prevent="$refs.menu.openContextMenu">
```

Comment:
Why is this event needed?

Reply:
That is the right click event that triggers opening the menu.

I had to put it on all 3 of these elements because $refs was undefined on the section and body tags.